penpot/penpot
# :tada: Add thumbnail renderer service

### How to test it:

1. Create multiple files with a lot of elements in the first frame or use huge files.
2. Go to the dashboard and check that thumbnails are rendered as bitmaps
3. [OPTIONAL] Check render times and main thread lockings on devtools

> NOTE: Doing the same steps without the thumbnail renderer should "lock" the main thread while thumbnails are being rendered.
